apps/store/extensions/app/social-reviews/themes/store/js/lib/handlebars.js

```diff
@@ -77,6 +77,16 @@ Handlebars.registerHelper('if', function(context, options) {
   }
 });
 
+Handlebars.registerHelper('if_equal', function (lvalue, rvalue, options) {
+    if (arguments.length < 3)
```

Comment:
Thank you, I have fixed this issue in following [commit](https://github.com/wso2/carbon-store/pull/643/commits/358a87f6fa0ffb067480daf1fe3e463bed87080e)